core/types/src/quote.rs

```diff
@@ -394,4 +396,208 @@ mod test {
         report.body.misc_select = 3;
         assert_eq!(info.report(), Report::from(report));
     }
+
+    #[test]
+    fn ct_eq_quoting_enclave_report_info() {
+        let nonce = sgx_quote_nonce_t { rand: [1u8; 16] };
+        let app_enclave_target_info = sgx_target_info_t {
+            mr_enclave: Default::default(),
+            attributes: Default::default(),
+            reserved1: [1u8; 2],
+            config_svn: 0,
+            misc_select: 0,
```

Comment:
:question: I think we can come back later and decide, but I wonder if the reserved fields should be excluded

Reply:
Why should it be excluded?

Reply:
Often reserved fields are for future expansion(and memory alignment). Often times they're whatever happened to be on the stack for the structure.